internal/config.go

```diff
@@ -168,7 +170,8 @@ var (
 	}
 
 	PGDefaultSettings = map[string]string{
-		PgWalSize: "16",
+		PgWalSize:        "16",
+		PgbackrestStanza: "main",
```

Comment:
I think that it is better to respect the original case, i.e. `PgBackRestStanza`.

Reply:
Rename PgbackrestStanza -> PgBackRestStanza